warehouse/src/table_update/main.py

```diff
@@ -118,13 +124,19 @@ def get_products(order_id: str) -> List[dict]:
     return products
 
 
+@metrics.log_metrics
```

Comment:
This function doesn't produce metric, should this be moved to the `parse_record` function?

Would this work as expected if we had lots of records (since `table_update` can get a large batch of records from DynamoDB in one go)? You can test that behaviour by using `make tests-perf`.

Reply:
That decorator expects the function to have two parameters, and is meant to be used with the [lambda handler](https://github.com/awslabs/aws-lambda-powertools-python/blob/develop/aws_lambda_powertools/metrics/metrics.py#L140).  

Lots of records should work, I did test with tests-perf